examples/ex9.cpp

```diff
@@ -16,6 +16,8 @@
 //    ex9 -m ../data/disc-nurbs.mesh -p 2 -r 3 -dt 0.005 -tf 9
 //    ex9 -m ../data/periodic-square.mesh -p 3 -r 4 -dt 0.0025 -tf 9 -vs 20
 //    ex9 -m ../data/periodic-cube.mesh -p 0 -r 2 -o 2 -dt 0.02 -tf 8
+//    ex9 -m ../data/periodic-square.msh -p 0 -r 2 -dt 0.005 -tf 2
+//    ex9 -m ../data/periodic-cube.msh -p 0 -r 1 -o 2 -tf 2 -no-vis -paraview
```

Comment:
The `-no-vis` option causes problems in autotest (because it is added there automatically). Can we remove it?

```suggestion
//    ex9 -m ../data/periodic-cube.msh -p 0 -r 1 -o 2 -tf 2 -paraview
```

Reply:
Yes, we can remove it. The reason `no-vis` is specified there is because none of the periodic surfaces are visualized in GLVis — it could be potentially confusing for a user to run one of the sample runs, and then have nothing show up in the visualization window.

Perhaps the periodic boundaries need to be added as "fake" boundary elements for visualization? I don't remember how that works.

Reply:
I'd prefer to "fix" the GLVis visualization if possible, maybe @mlstowell can help with this?

The alternative is for OptionParser not to complain if you have the same option twice, the current autotest error is `Option --visualization or --no-visualization provided multiple times`.